lang/java/reef-common/src/main/java/org/apache/reef/driver/evaluator/EvaluatorType.java

```diff
@@ -30,6 +30,9 @@
    * Indicates an Evaluator that runs on the CLR.
    */
   CLR,
+  /** Indicates an Evaluator that runs on a local thread.
+   */
+   LOCAL,
```

Comment:
Currently we have Evalautor written in Java and .Net. EveluatorType indicates what runtime env. that the Evalautor is running on, Java Evaluator runs on JVM and C# Evaluator runs on CLR. Both can run either locally or on YARN. Not sure what LOCAL means here? Does the thread model only apply for local runtime?